security/pkg/pki/ca/controller/secret.go

```diff
@@ -60,7 +60,10 @@ const (
 type SecretController struct {
 	ca      ca.CertificateAuthority
 	certTTL time.Duration
-	core    corev1.CoreV1Interface
+	// Length of the grace period for the certificate rotation.
+	gracePeriod time.Duration
```

Comment:
Should we always just calculate the gracePeriod from the ratio, rather than configuring both the ttl and the gracePeriod independently?